src/components/info/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import InfoSection from './info-section';
+import { Section, Link } from './info.style';
+
+export default function Info() {
+  return (
+    <Section>
+      <InfoSection id="how-it-works" headline="How it works">
```

Comment:
Your previous implementation before the code review was good as well btw.

Reply:
The code you showed was much cleaner and easier to understand, and that's something I'm aiming for :)